x-pack/plugins/canvas/public/components/datasource/__stories__/datasource_component.stories.tsx

```diff
@@ -70,6 +70,7 @@ storiesOf('components/datasource/DatasourceComponent', module)
       setPreviewing={action('setPreviewing')}
       isInvalid={false}
       setInvalid={action('setInvalid')}
+      renderError={action('renderError')}
```

Comment:
Is this necessary for the image stuff, or does this belong with error renderer?

Reply:
After adding async libs loading and storybook snapshots to support it, I've faced the problem with mistakes at `datasource_component.tsx` and related `storybook`. While running a test of `snapshots`, It is coming to the part, where `renderError` is required to be defined.  Otherwise, it was not passing `jest` tests.